lib/litani_report.py

```diff
@@ -74,9 +74,15 @@ def render(self, gnu_file, out_file=None):
 # ______________________________________________________________________________
 
 
-@dataclasses.dataclass
 class PipelineDepgraphRenderer:
-    pipe: dict
+    def __init__(self, pipe):
+        self.pipe = pipe
```

Comment:
`self.pipe` is no longer being used. It should be removed. There's no need to pass anything in the constructor any more, since there's already a pipe passed into `render`.